data/build.gradle

```diff
@@ -5,15 +5,21 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'realm-android'
 
 android {
+
+    compileOptions {
+        sourceCompatibility = 1.7
+        targetCompatibility = 1.7
+    }
+    buildToolsVersion = '29.0.2'
     def globalConfiguration = rootProject.extensions.getByName("ext")
+    compileSdkVersion = 29
 
-    compileSdkVersion globalConfiguration.getAt("androidCompileSdkVersion")
     defaultConfig {
-        minSdkVersion globalConfiguration.getAt("androidMinSdkVersion")
```

Comment:
please use the variables already created.. just update the variable values ;)

Reply:
done